src/OperationalInsights/OperationalInsights/Models/PSKeyVaultProperties.cs

```diff
@@ -18,29 +18,36 @@ namespace Microsoft.Azure.Commands.OperationalInsights.Models
 {
     public class PSKeyVaultProperties
     {
-        public PSKeyVaultProperties(string keyVaultUri = default, string keyName = default, string keyVersion = default, int keyRsaSize = default)
+        public PSKeyVaultProperties(string keyVaultUri, string keyName, string keyVersion)
         {
             KeyVaultUri = keyVaultUri;
             KeyName = keyName;
             KeyVersion = keyVersion;
-            KeyRsaSize = keyRsaSize;
         }
 
-        public static PSKeyVaultProperties CreateProperties(string keyVaultUri = default, string keyName = default, string keyVersion = default, int keyRsaSize = default)
+        /// <summary>
+        /// Creates an instance of PSKeyVaultProperties
+        /// </summary>
+        /// <param name="keyVaultUri"></param>
+        /// <param name="keyName"></param>
+        /// <param name="keyVersion"></param>
+        /// <returns>PSKeyVaultProperties object ,if no value was passed a null will be returned</returns>
+        public static PSKeyVaultProperties CreateProperties(string keyVaultUri = null, string keyName = null, string keyVersion = null)
         {
-            if (keyVaultUri == null && keyName == null && keyVersion == null && keyRsaSize == 0)
+            if (keyVaultUri == null && keyName == null && keyVersion == null)
             {
                 return null;
             }
 
-            return new PSKeyVaultProperties(keyVaultUri, keyName, keyVersion, keyRsaSize);
+            return new PSKeyVaultProperties(keyVaultUri, keyName, keyVersion);
         }
 
         public PSKeyVaultProperties(KeyVaultProperties kv)
         {
             this.KeyVaultUri = kv.KeyVaultUri;
```

Comment:
kv.KeyVaultUri, kv.KeyVersion, kv.KeyName can be empty or null:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/operationalinsights/Microsoft.Azure.Management.OperationalInsights/src/Generated/Models/KeyVaultProperties.cs#L40

Reply:
@VeryEarly  not sure what you meant here